japanese/lookup.py

```diff
@@ -5,22 +5,39 @@
 # Dictionary lookup support.
 #
 
-import urllib, re
+import re
 from anki.hooks import addHook
 from aqt import mw
 from aqt.qt import *
 from aqt.utils import showInfo
 
+# Backwards compatibility
+try:
+    from urllib import quote  # Python 2.X
+except ImportError:
+    from urllib.parse import quote  # Python 3+
+try:
+    Copy = QWebEnginePage.Copy # Python 3+
+except NameError:
+    Copy = QWebPage.Copy # Python 2.X
+try:
+    setUrl = QUrl.setEncodedUrl # Python 2.X
+except AttributeError:
+    setUrl = QUrl.setUrl # Python 3+
+
 class Lookup(object):
 
     def __init__(self):
         pass
 
     def selection(self, function):
         "Get the selected text and look it up with FUNCTION."
+        #TODO This doesn't seem to work...
```

Comment:
I don't remember why it's like that - it's possible it just wasn't clear at the time how to get the selection otherwise.

Does the solution you provided work on both 2.0 and 2.1? I thought by your comment it may not be working, but perhaps you were just referring to the commented text? If it's just that the commented text doesn't work, feel free to snip it out completely.